### DISCUSSION TITLE
Detail of AlphaPulldown dependencies

### DISCUSSION THREAD

**Discussion by nmarounina** [KosinskiLab/AlphaPulldown#215] [Show and tell]
Hello everyone,

I am working on an HPC cluster that does not support conda, therefore, I was going to move forward and install the software by hand. I have found the ./conda/environment.yml file that looks like is going to help me a lot, and I was wondering which dependencies are strictly mandatory to run the software and which one are optional ?

**Comment by dingquanyu** [comment]
Hi,

Thanks for your interest in our package. Essentially, the packages listed in the instruction commands are required plus when you install AlphaPulldown from pip, pip also looks for necessary packages and will install them if they haven't been installed. In this case, you could refer to the config.cfg and check which python dependencies are listed. Hope it helps.

Yours, 
Dingquan
